src/sentry/api/serializers/models/organization_member/base.py

```diff
@@ -88,6 +89,10 @@ def serialize(
             "dateCreated": obj.date_added,
             "inviteStatus": obj.get_invite_status_name(),
             "inviterName": obj.inviter.get_display_name() if obj.inviter else None,
+            "allOrgRoles": serialize(
```

Comment:
It will be better to rename this to `orgRole<something>` so it's right next to `orgRole`.